karate-tests/src/test/java/mocks/sandbox/get-patient-search.js

```diff
@@ -149,14 +150,20 @@ if (request.pathMatches('/Patient') && request.get) {
 
   if (validateHeaders(request) && validateQueryParams(request)) {
     if (fuzzyMatch) {
-      if (family === 'Garton' && (given[0]) === 'Bill' && birthDate[0] === '1946-06-23') {
+      if (family === 'Blogs' && given[0] === 'Joe' && birthDate[0] === '1955-11-05') {
         response.body = timestampBody(BILL_GARTON_SEARCHSET)
```

Comment:
Is this correct?

Reply:
perhaps rename or redefine HISTORIC_JOE_BLOGS to make it clear what is happening here?

Reply:
The idea behind it is the request is has the fuzzy match flag set, and is matching on the historic name of Joe Blogs. 
I've renamed it to JOE_BLOGS_HISOTRIC_NAME_SEARCHSET.